#VRML_SIM R2019a utf8
WorldInfo {
}
Viewpoint {
  orientation -0.6246904514368393 -0.7589976471519615 -0.18353313461450038 0.7705265039223199
  position -1.1318654215854682 1.5131905172546942 2.5455614581848507
}
TexturedBackground {
}
TexturedBackgroundLight {
}
Floor {
}
Robot {
  translation 0.1 0.07999999999999997 0.1
  children [
    DEF GR_BODY Group {
      children [
        DEF ROBOT_BODY Shape {
          appearance PBRAppearance {
            baseColor 0.9372549019607843 0.1607843137254902 0.1607843137254902
          }
          geometry DEF G_BODY Box {
            size 0.25 0.07 0.1
          }
        }
        Transform {
          translation 0.11 -0.05 0
          children [
            Shape {
              appearance PBRAppearance {
                roughness 1
                metalness 0
              }
              geometry Sphere {
                radius 0.015
              }
            }
          ]
        }
      ]
    }
    DEF WHEEL1 HingeJoint {
      jointParameters HingeJointParameters {
        axis 0 0 1
        anchor -0.05 0 0.06
      }
      device [
        PositionSensor {
          name "encoder1"
        }
        RotationalMotor {
          name "motor_right"
          maxVelocity 52.35
          maxTorque 0.59
        }
      ]
      endPoint DEF WHEEL1 Solid {
        translation -0.05 0.0025 0.060000000000000005
        rotation 1 0 0 1.57
        children [
          DEF S_WHEEL1 Shape {
            appearance PBRAppearance {
              baseColor 0 0 0
              roughness 1
              metalness 0
            }
            geometry DEF G_WHEEL Cylinder {
              height 0.02
              radius 0.075
            }
          }
        ]
        boundingObject USE G_WHEEL
        physics Physics {
        }
      }
    }
    DEF WHEEL2 HingeJoint {
      jointParameters HingeJointParameters {
        axis 0 0 1
        anchor -0.05 0 -0.06
      }
      device [
        RotationalMotor {
          name "motor_left"
          maxVelocity 52.3598
          maxTorque 0.59
        }
      ]
      endPoint Solid {
        translation -0.05161149370379927 0.0010565499530129716 -0.06
        rotation 1 0 0 1.57
        children [
          DEF WH_SHAPE2 Shape {
            appearance PBRAppearance {
              baseColor 0 0 0
              roughness 1
              metalness 0
            }
            geometry Cylinder {
              height 0.02
              radius 0.075
            }
          }
        ]
        name "solid(1)"
        boundingObject USE WH_SHAPE2
        physics Physics {
        }
      }
    }
    DistanceSensor {
      translation 0.125 0 0
      children [
        Shape {
          appearance PBRAppearance {
            baseColor 0.45098039215686275 0.8235294117647058 0.08627450980392157
          }
          geometry Box {
            size 0.02 0.01 0.01
          }
        }
      ]
      name "distance_sensor"
      lookupTable [
        0 0 0
        0.02 0 0
        4 1023 0
      ]
      resolution 0.3
    }
  ]
  boundingObject USE GR_BODY
  physics Physics {
    density -1
    mass 3
  }
  controller "u2_Exam_Navarro_Esther"
}
Solid {
  translation 0.71 0.1 0.15000000000000002
  children [
    Shape {
      appearance PBRAppearance {
      }
      geometry DEF G_BOX Box {
        size 0.1 0.2 0.2
      }
    }
  ]
  boundingObject USE G_BOX
  physics Physics {
  }
}
